src/IRremoteESP8266.h

```diff
@@ -766,7 +774,7 @@
      DECODE_AMCOR || DECODE_DAIKIN152 || DECODE_MITSUBISHI136 || \
      DECODE_MITSUBISHI112 || DECODE_HITACHI_AC424 || DECODE_HITACHI_AC3 || \
      DECODE_HITACHI_AC344 || DECODE_CORONA_AC || DECODE_SANYO_AC || \
-     DECODE_VOLTAS || DECODE_MIRAGE || DECODE_HAIER_AC176)
+     DECODE_VOLTAS || DECODE_MIRAGE || DECODE_HAIER_AC176 || DECODE_KELON)
```

Comment:
If not using state than this is not needed

Reply:
What do you mean? Doesn't this enable parsing the state from `stdAc::state_t`? I implemented it.